docs/usage.rst

```diff
@@ -44,6 +44,9 @@ The full documentation can be issued by running with ``--help``:
                             Use with -i to specify absolute path to a
                             `requirements.txt` you wish to use, else we'll look
                             for one in the current working directory.
+      -omit OMIT, --omit OMIT
```

Comment:
Remove `-omit` as now removed from `client.py`

Reply:
added a reminder in #535